tests/unit/DiffOp/Diff/DiffTest.php

```diff
@@ -258,7 +258,7 @@ public function testHasAssociativeOperations( Diff $diff, $hasAssocOps ) {
 	public function testConstructor( array $elements ) {
 		$arrayObject = new Diff( $elements );
 
-		$this->assertEquals( count( $elements ), $arrayObject->count() );
+		$this->assertCount( $arrayObject->count(), $elements );
```

Comment:
This is the wrong way round now I think.

`$arrayObject->count()` is the actual value, not the expected one

I'm confused by the test method name though - not sure how this assertion relates to the constructor

Reply:
Yes. You're right.